src/librustc_mir/monomorphize/collector.rs

```diff
@@ -1197,6 +1197,7 @@ fn collect_miri<'tcx>(
             }
         }
         GlobalAlloc::Function(fn_instance) => {
+            let fn_instance = fn_instance.polymorphize(tcx);
```

Comment:
Is this change necessary? - `create_fn_mono_item` and `should_codegen_locally` both invoke `polymorphize` IIRC.

Reply:
it's not :+1: